bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/read/ReaderExpressionHelper.java

```diff
@@ -108,6 +108,13 @@ public void write(int unquoted) throws IOException {
         writeNullCharacterBytes();
         return;
       }
+      if (unquoted < 0) {
```

Comment:
Shouldn't this be `> 127` ? unquoted is supposed to represent an unsigned byte here

Reply:
@igorbernstein2 No, this method accepts int but is invoked with a byte passed as the parameter:
https://github.com/googleapis/java-bigtable-hbase/blob/534288cfebf4732b7998c369a6e278581a64f758/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/FuzzyRowFilterAdapter.java#L75